packages/designsystem/src/stories/FormGeneric.story.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <Story title="FormGeneric" icon="bi:wrench-adjustable">
+    <Variant title="default">
+      <FormGeneric
+        @cancel="logEvent('cancel', $event)"
+        @submit="logEvent('submit', $event)"
+      >
+        <Field
+          v-slot="{ field, meta: { valid, touched } }"
+          v-model="name"
+          name="name"
+          type="input"
+          rules="required"
+          class="mb-4"
+          as="div"
+        >
+          <PrimeInput
+            v-focus
+            v-bind="field"
+            placeholder="name"
+            :required="true"
+            :class="{ 'p-invalid': !valid && touched }"
+          />
+          <ErrorMessage class="p-text-error" name="name" />
+        </Field>
+        <Field
+          v-slot="{ field }"
+          v-model="pseudo"
+          class="mb-4"
+          name="pseudo"
+          type="input"
+          as="div"
+        >
+          <PrimeInput v-bind="field" placeholder="pseudo" />
+        </Field>
+      </FormGeneric>
+    </Variant>
+    <Variant title="withFullButton">
+      <FormGeneric
+        :full-width-button="true"
+        @cancel="logEvent('cancel', $event)"
+        @submit="logEvent('submit', $event)"
+      >
+        <Field
+          v-slot="{ field, meta: { valid, touched } }"
+          v-model="name"
+          name="name"
+          type="input"
+          rules="required"
+          class="mb-4"
+          as="div"
+        >
+          <PrimeInput
+            v-focus
+            v-bind="field"
+            placeholder="name"
+            :required="true"
+            :class="{ 'p-invalid': !valid && touched }"
+          />
+          <ErrorMessage name="name" class="p-text-error" />
+        </Field>
+        <Field
+          v-slot="{ field }"
+          v-model="pseudo"
+          class="mb-4"
+          name="pseudo"
+          type="input"
+          as="div"
+        >
+          <PrimeInput v-bind="field" placeholder="pseudo" />
+        </Field>
+      </FormGeneric>
+    </Variant>
+  </Story>
+</template>
+
+<script setup lang="ts">
+import FormGeneric from '../components/FormGeneric/FormGeneric.vue'
+import PrimeInput from 'primevue/inputtext'
+import { Field, ErrorMessage } from 'vee-validate'
+import { ref } from 'vue'
+import { logEvent } from 'histoire/client'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const emit = defineEmits<{
+  (e: 'submit', name: string, pseudo: string | null): void
+}>()
+
+const name = ref()
+const pseudo = ref()
+</script>
+
+<!--<docs lang="md">-->
+<!--### FormGeneric-->
```

Comment:
Mmm, This doc seems ok ?

Why did you comment ?

Reply:
I forget to restore it :)